.github/workflows/pr_verification.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: PR verification workflow
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
We'd want this to run for any PR, not just those targeting main, right? I know it's rare for our project, but in the past we had rather large features get PRs into their own feature branch. Plus we have the individual CP maintenance branches.

Reply:
Good point. Removed branch filter.